source/common/http/headers.h

```diff
@@ -113,6 +113,9 @@ class CustomHeaderValues {
 
   struct {
     const std::string AcceptEncoding{"Accept-Encoding"};
+    const std::string Accept{"Accept"};
+    const std::string UserAgent{"User-Agent"};
+    const std::string ContentType{"Content-Type"};
```

Comment:
They are not used in this PR right now. I probably should use them for the tests where I had the raw string `Accept`. The `CustomHeaders` is intended for extensions per the PR description of #11791. My understanding of the `VaryValues` struct is intended to contain the possible values of the `Vary` field for reference purposes. So when developers type `Headers::get().VaryValues` it auto-completes. I'm not sure whether using it for tests is sufficient to justify its existence here. If not, I'll remove these changes.

Reply:
Suggest pushing this out to the next PR that actually uses them (assuming that'd be easy :)

Moreover I am wondering if we should be declared in an extension-specified header-file rather than a core header file. Otherwise the core grows to the aggregate size of all extensions.

Reply:
I'll remove these changes from the PR. We can create a task to detach these custom headers from the core.